src/diag/diag_process.cpp

```diff
@@ -307,6 +308,53 @@ void Diag::ProcessStats(int aArgCount, char *aArgVector[], char *aOutput, size_t
     AppendErrorResult(error, aOutput, aOutputMaxLen);
 }
 
+void Diag::ProcessReceive(int aArgCount, char *aArgVector[], char *aOutput, size_t aOutputMaxLen)
```

Comment:
Make receive/sleep as sub commands of `radio`?
USAGE:
```sh
> diag radio sleep
> diag radio receive
```

Reply:
In README.md there is "diag sleep" but not implemented. From the usage: "Enter radio sleep mode.", actually this command is just what we need here. Do we need to follow the command style in README, then update the README accordingly?

Reply:
@yuzhyang The “diag sleep” cmd was implemented, but removed by #2378 to make it as a platform specific feature, but I forgot to delete it from README. 
Please help to delete the "sleep" cmd, and add "radio sleep/receive" cmd.

Reply:
The commands "send, power, channel, repeat" all seems to be the radio behaviors. Is this OK to keep it short and the function could be single as well?

Reply:
The "diag sleep" may leads to some misunderstanding: radio sleep or MCU sleep (or even chip sleep while radio on RX), that's why we remove the "diag sleep" feature to be platform specific. "diag radio sleep" would be better to describe the behavior.